libs/minilibx/events.md

```diff
@@ -141,13 +141,29 @@ int	main(void)
 Here we register to the `KeyPress` event with the according `KeyPressMask`. Now
 whenever we press a key, the window will be closed.
 
+### Prototype of event functions
+
+Event functions have different prototype depending of the hooking event.
+
+| Hooking event | code | Prototype            |
+| ------------- | :--: | -------------------- |
+| ON_KEYDOWN    |   2  | `keydown(int keycode, t_vars *vars)`   |
+| ON_KEYUP*     |   3  | `keyup(int keycode, t_vars *vars)` |
+| ON_MOUSEDOWN* |   4  | `mousedown(int keycode, int x, int y, t_vars *vars)` |
+| ON_MOUSEUP    |   5  | `mouseup(int keycode, int x, int y, t_vars *vars)` |
+| ON_MOUSEMOVE  |   6  | `mousemove(int x, int y, t_vars *vars)` |
+| ON_EXPOSE*    |  12  | `expose(t_vars *vars)` |
+| ON_DESTROY    |  17  | `close(t_vars *vars)` |
```

Comment:
Can you sort this alphabetically and can you denote the prototypes as a function pointer?

Reply:
I ordered by id asc